tw-base-utils/src/main/java/com/transferwise/common/baseutils/concurrency/CountingThreadFactory.java

```diff
@@ -12,9 +12,7 @@ public class CountingThreadFactory implements ThreadFactory {
   public CountingThreadFactory(String groupName) {
     this.groupName = groupName;
 
-    SecurityManager s = System.getSecurityManager();
-    group = (s != null) ? s.getThreadGroup() : Thread.currentThread().getThreadGroup();
-
+    group = Thread.currentThread().getThreadGroup();
```

Comment:
SecurityManager will be removed anyway soon. And we are not running anything in application servers.